canvasapi/assignment.py

```diff
@@ -205,6 +227,60 @@ def get_submissions(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def publish_provisional_grades(self, **kwargs):
+        """
+        Publish the selected provisional grade for all submissions to an assignment.
+        Use the “Select provisional grade” endpoint to choose which provisional grade to publish
+        for a particular submission.
+
+        Students not in the moderation set will have their one
+        and only provisional grade published.
+
+        WARNING: This is irreversible. This will overwrite existing grades in the gradebook.
+
+        :calls: `POST /api/v1/courses/:course_id/assignments/:assignment_id/provisional_grades
+        /publish \
+        <https://canvas.instructure.com/doc/api/all_resources.html#method.provisional_grades.publish>`_
+        :param course_id: ID of specified course
+        :type course_id: int
+        :param assignment_id: ID of specified assignment
+        :type assignment_id: int
```

Comment:
These aren't parameters that the user inputs; they're already handled internally by CanvasAPI. remove these docstrings.